src/index.html

```diff
@@ -237,22 +237,30 @@ <h2>SKILLS</h2>
       <div>
         <section>
           <h2>QUICK ID</h2>
-          <ul>
-            <li><b>Phone</b><a href="tel:123-456-7890">123-456-7890</a></li>
-            <li>
-              <b>Email</b><a href="mailto:info@mysite.com">info@mysite.com</a>
-            </li>
-            <li>
-              <b>Website</b
-              ><a href="http://www.wix.com" target="_blank">www.wix.com</a>
-            </li>
-            <li>
-              <b>Address</b>
-              <address>
-                500 Terry Francois Street San Francisco, CA 94158
-              </address>
-            </li>
-          </ul>
+          <table id="table1">
```

Comment:
Thanks for submitting the PR promptly. However we prefer you to submit it when finished all assignment requirements.
We normally won't approve partially finished PR in industry standard. 
I can start review it a little bit, but please finish everything before submitting a PR next time. Thanks

Reply:
Ok, thank you